CODEOWNERS

```diff
@@ -594,6 +594,7 @@ Makefile* @cilium/build
 /pkg/resiliency @cilium/sig-agent
 /pkg/revert/ @cilium/sig-agent
 /pkg/safeio @cilium/sig-agent
+/pkg/safenetlink @cilium/sig-datapath
```

Comment:
netlink afaik is Linux (& FreeBSD) specific, could have it under `pkg/datapath/linux/safenetlink`?

Reply:
So it is imported by the non-Linux builds of Cilium CLI (which is why I had to add the non-Linux stubs). But yeah, it is overall very Linux specific, we could move it.

Reply:
Done